reports/historical/_gmmp_2015_importer.py

```diff
@@ -304,6 +317,18 @@ def import_6(self, sheet_data):
             row_heading_col=2,
         )
 
+        for year, country_data_per_media_per_region in all_data.items():
+            if year != 2010:
+                # 2015 data introduced the media rows.
+                country_data_per_media_per_region = {**country_data_per_media_per_region['print, radio, television'],
+                                                        **country_data_per_media_per_region['internet, twitter']}
+            year_data = {}
+            for continent, continent_data in country_data_per_media_per_region.items():
+                year_data[continent] = {}
+                for _, gender in continent_data.items():
+                    data = self.combine_p_n(continent_data)
+                    year_data[continent].update(data)
```

Comment:
Seems like we're not using anything from this loop i.e. `_` is ignored and `gender` is not used anywhere. Why do we need it?

Reply:
Yeah. I was testing out something. I forgot to remove this.